### ISSUE TITLE
Dead lib?

### ISSUE THREAD

**Issue by liamzee** [zsedem/haskell-cpython#19]
Is there still any interest on this project? I'm currently having issues getting this working on 9.4, but that's sort of to be expected given that this library saw its last update in 2021.

Warning: The package list for 'hackage.haskell.org' is 19 days old.
Run 'cabal update' to get the latest list of available packages.
Warning: The package list for 'hackage.haskell.org' is 19 days old.
Run 'cabal update' to get the latest list of available packages.
Resolving dependencies...
Error: cabal-3.8.1.0.exe: Could not resolve dependencies:
[__0] trying: cpython-3.5.1 (user goal)
[__1] rejecting: cpython:-usepython38 (conflict: pkg-config package
python3-any, not found in the pkg-config database)
[__1] rejecting: cpython:+usepython38 (conflict: pkg-config package
python-3.8-any, not found in the pkg-config database)
[__1] fail (backjumping, conflict set: cpython, cpython:usepython38)
After searching the rest of the dependency tree exhaustively, these were the
goals I've had most trouble fulfilling: cpython, cpython:usepython38

**Comment by zsedem** [comment]
yeah pretty much dead at the moment. Also I have to say I never tried it on Windows.
